lib/pbench/server/unpack_tarballs.py

```diff
@@ -129,4 +129,4 @@ def report(self, prog: str, result_string: str):
             try:
                 report.post_status(self.config.timestamp(), "status", reportfp.name)
             except Exception:
-                pass
+                self.logger.exception("{}: failure posting report", prog)
```

Comment:
Are you sure you want `exception()` here?  I.e., is the traceback really interesting, and do we really want "error" level?  Or, should this be `info()` with the stringized exception message?

Reply:
There's a fair amount of code under `post_status`; so if something happens, I think the stack trace is likely to be handy.